src/docfx/build/Build.cs

```diff
@@ -16,7 +16,7 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Repo
         {
             var errors = new List<Error>();
 
-            var (configErrors, config) = Config.Load(docsetPath, options);
+            var (configErrors, config) = LocalizationConvention.GetBuildConfig(docsetPath, options);
```

Comment:
While building `en-us` repo, we also need to get the build config from `LocalizationConvention`?